### ISSUE TITLE
Expose Data To Pan Background Widget

### ISSUE THREAD

**Issue by jtkeyva** [alihadi5125/smooth_compass#1]
This is great thanks! 

Is there any way to use it to pan a widget? Seems doable, but the data needs to be exposed. Any way we can get rotation data exposed that can be passed to another widget like an image inside of Interactive Viewer
https://api.flutter.dev/flutter/widgets/InteractiveViewer-class.html

Any insight appreciated.
Thanks!

**Comment by fahadshabeer** [comment]
yes you can use compass stream for it, you can get the whole compass data from the stream and use it anywhere
